Code/Source/RobotImporter/URDF/CollidersMaker.cpp

```diff
@@ -0,0 +1,341 @@
+            if (valueStorage.empty())
+            {
+                AZ_Error("CollisionMaker", false, "Error loading collider. Invalid value storage.");
+                return;
+            }
+
+            auto view = AZ::SceneAPI::Containers::MakeDerivedFilterView<AZ::SceneAPI::DataTypes::ISceneNodeGroup>(valueStorage);
+            if (view.empty())
+            {
+                AZ_Error("CollisionMaker", false, "Error loading collider. Invalid node views.");
+                return;
+            }
+
+            // Select all nodes for both visual and collision nodes
+            for (AZ::SceneAPI::DataTypes::ISceneNodeGroup& mg : view)
+            {
+                AZ::SceneAPI::Utilities::SceneGraphSelector::SelectAll(scene->GetGraph(), mg.GetSceneNodeSelectionList());
+            }
+
+            // Update scene with all nodes selected
+            AZ::SceneAPI::Events::ProcessingResultCombiner result;
+            AZ::SceneAPI::Events::AssetImportRequestBus::BroadcastResult(
+                result,
+                &AZ::SceneAPI::Events::AssetImportRequest::UpdateManifest,
+                *scene,
+                AZ::SceneAPI::Events::AssetImportRequest::ManifestAction::Update,
+                AZ::SceneAPI::Events::AssetImportRequest::RequestingApplication::Editor);
+
+            if (result.GetResult() != AZ::SceneAPI::Events::ProcessingResult::Success)
+            {
+                AZ_Error("CollisionMaker", false, "Scene updated");
+                return;
+            }
+
+            auto assetInfoFilePath = azMeshPath;
+            assetInfoFilePath.Native() += ".assetinfo";
+            AZ_Printf("CollisionMaker", "Saving collider manifest to %s", assetInfoFilePath.c_str());
+            scene->GetManifest().SaveToFile(assetInfoFilePath.c_str());
+
+            bool assetFound = false;
+            AZ::Data::AssetInfo assetInfo;
+            AZStd::string watchDir;
+            AzToolsFramework::AssetSystemRequestBus::BroadcastResult(
+                assetFound,
+                &AzToolsFramework::AssetSystem::AssetSystemRequest::GetSourceInfoBySourcePath,
+                azMeshPath.c_str(),
+                assetInfo,
+                watchDir);
+
+            // Set export method to convex mesh
+            auto readOutcome = AZ::JsonSerializationUtils::ReadJsonFile(assetInfoFilePath.c_str());
+            if (!readOutcome.IsSuccess())
+            {
+                AZ_Error("CollisionMaker", false, "Could not read %s", assetInfoFilePath.c_str());
+                return;
+            }
+            rapidjson::Document assetInfoJson = readOutcome.TakeValue();
+            auto manifestObject = assetInfoJson.GetObject();
+            auto valuesIterator = manifestObject.FindMember("values");
+            if (valuesIterator == manifestObject.MemberEnd())
+            {
+                AZ_Error("CollisionMaker", false, "Invalid json file: %s", assetInfoFilePath.c_str());
```

Comment:
Should add a message indication thaat 'values' was not found.
```suggestion
                AZ_Error("CollisionMaker", false, "Invalid json file: %s (Missing 'values' node)", assetInfoFilePath.c_str());
```